README.md

```diff
@@ -308,8 +308,70 @@ just added.
 ## API
 
 <generated_api_start />
+#### The Jest global environment
+
+In your test files, Jest puts each of these methods and objects into the global environment. You don't have to require or import anything to use them.
+
+  - `afterEach(fn)`
+  - `beforeEach(fn)`
+  - [`check`](https://facebook.github.io/jest/docs/api.html#check)
```

Comment:
Hrmmm. Is anything else in `jest-check` or just these two things? I hate to leave something in there, but undocumented - people will be reading other peoples' code, see this function, go to our docs to see what it does, and be annoyed that we deleted the docs.